File: compiled_lang/go/c20.arithmetic.go
package main

import (
	"fmt"
	"math"
)

func main() {
	radius := 3
	area := math.Pi * math.Pow(float64(radius), 2)

	fmt.Printf("The area of a circle (radius=%d) is: %v.\n", radius, area)
}

File: compiled_lang/cpp/c20.arithmetic.cpp
#include <cmath>
#include <iostream>

int main() {
    // acos(-1.0), not the M_PI macro - M_PI isn't standard C++ and needs
    //  _USE_MATH_DEFINES before <cmath> on MSVC, so this stays portable
    const double pi = std::acos(-1.0);
    int radius = 3;
    double area = pi * std::pow(radius, 2);

    std::cout << "The area of a circle (radius=" << radius << ") is: " << area << "." << std::endl;

    return 0;
}
